src/core/lib/iomgr/tcp_server_windows.cc

```diff
@@ -100,6 +100,9 @@ static grpc_error* tcp_server_create(grpc_closure* shutdown_complete,
                                      const grpc_channel_args* args,
                                      grpc_tcp_server** server) {
   grpc_tcp_server* s = (grpc_tcp_server*)gpr_malloc(sizeof(grpc_tcp_server));
+  if (s == nullptr) {
+    return GRPC_ERROR_OOM;
+  }
```

Comment:
if gpr_malloc() can not alloc memory, it will raise an abort();

Reply:
@weiyongji updated with no guards.